enterprise/cluster/src/main/java/org/neo4j/cluster/protocol/atomicbroadcast/multipaxos/context/HeartbeatContextImpl.java

```diff
@@ -141,6 +141,14 @@ public void notify( HeartbeatListener listener )
     @Override
     public void suspicions( InstanceId from, Set<InstanceId> suspicions )
     {
+        // Ignoring this message if it is the first received from a failed instance.
+        if ( isFailed( from ) )
```

Comment:
Comment says that we ignore suspicions if it is the first message we get from a failed instance.
How is the 'first received' condition checked/ensured?